classMemory.js

```diff
@@ -0,0 +1,26 @@
+(function() {
+    "use strict";
+
+    function Memory() {
+        this.num = 0;
+    }
+
+    Memory.prototype.mPlus = function(number) {
+        this.num += Number(number);
+    }
+
+    Memory.prototype.mMinus = function(number) {
+        this.num += Number(-number);
+    }
+
+    Memory.prototype.mRecall = function() {
+        return this.num;
+    }
+
+    Memory.prototype.mClear = function() {
+        this.num = 0;
+        return this.num;
```

Comment:
There is no point in returning anything here, you know it will be zero, since you reseted it.